### PR TITLE
CV2-3821: Only run init on Live when manually invoked

### PR DESCRIPTION
**Pull request by computermacgyver**
## Description
This avoids race conditions when multiple workers are running and requires migrations to be run manually for the time being.

Reference: CV2-3821

## How has this been tested?
To be tested locally still

## Have you considered secure coding practices when writing this code?
We'll need to remember to actually run migrations on Live as a post-deploy task

### CODE REVIEW

File: manage.py [hunk 1]
```diff
@@ -260,8 +260,16 @@ def run_video_matcher():
   VideoMatcher.start_server()
 
 @manager.command
-def init():
+def live_init():
+  init(True)
+
+@manager.command
+def init(run_in_live=False):
```
[the remark this answers is not in the document]
Comment (skyemeedan) [reply manage.py:267]: I agree with code climate, is the caller expecting a database object or a boolean?

File: app/main/config.py [hunk 2]
```diff
@@ -4,6 +4,7 @@
 basedir = os.path.abspath(os.path.dirname(__file__))
 
 class Config:
+  ENVIRONMENT = os.getenv("DEPLOY_ENV", "local")
   SECRET_KEY = os.getenv('SECRET_KEY', 'my_precious_secret_key')
```
Comment (skyemeedan) [inline app/main/config.py:8]: what is this secret key for?

**Review by skyemeedan — APPROVED** [review]

File: app/main/config.py (hunk 2 above)
Comment (computermacgyver) [reply app/main/config.py:8]: No idea and unrelated to this PR, but I am curious :joy:

File: manage.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (computermacgyver) [reply manage.py:267]: These are functions invoked on the terminal; so, the caller is not expecting anything. I agree, however, that this is a point to be consistent. I personally like returning a boolean indicating if the function ran as expected and will implement this.

**Pull request closed without merging**